benchmarks/datasets/logs/queries/pg_search/bucket-numeric-filter.sql

```diff
@@ -6,3 +6,6 @@ SELECT * FROM paradedb.aggregate(index=>'benchmark_logs_idx', query=>paradedb.te
 
 -- aggregate without mvcc
 SELECT * FROM paradedb.aggregate(index=>'benchmark_logs_idx', query=>paradedb.term('message', 'research'), agg=>'{"buckets": { "terms": { "field": "severity" }}}', solve_mvcc=>false);
+
+-- aggregate custom scan
+SET paradedb.enable_aggregate_custom_scan TO on; SELECT severity, COUNT(*) FROM benchmark_logs WHERE message @@@ 'research' GROUP BY severity ORDER BY severity;
```

Comment:
According to https://github.com/paradedb/paradedb/pull/2909#pullrequestreview-3064315460, this is slower than `aggregate with mvcc`, which is unexpected: is the aggregate being defined in a different way than:
```sql
SELECT * FROM paradedb.aggregate(index=>'benchmark_logs_idx', query=>paradedb.term('message', 'research'), agg=>'{"buckets": { "terms": { "field": "severity" }}}', solve_mvcc=>true);
```
?

Reply:
I see that. Thanks for pointing it out. These are now two completely different beasts. Added an example in b29eaf1d0c916f87ce9403cf49d16165f6a6ad3e

I will keep this on mind, and we should get back to it for perf improvements.

Reply:
Oh, BTW, the queries are not equivalent, as the ` aggregate with mvcc` doesn't have ORDER BY.